packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -193,12 +217,26 @@ class HttpRouteSpec extends RouteSpec {
       match: {
         prefix: prefixPath,
       },
+      timeout: this.timeout ? this.renderTimeout(this.timeout): undefined,
     };
     return {
       httpRouteSpec: this.protocol === Protocol.HTTP ? httpConfig : undefined,
       http2RouteSpec: this.protocol === Protocol.HTTP2 ? httpConfig : undefined,
     };
   }
+
+  private renderTimeout(timeout: HttpTimeout): CfnRoute.HttpTimeoutProperty {
+    return ({
```

Comment:
Why did you add the `( ... )` around the return?

Reply:
copy pasta from elsewhere, ack